github/github_test.go

```diff
@@ -911,3 +911,41 @@ func TestBasicAuthTransport_transport(t *testing.T) {
 		t.Errorf("Expected custom transport to be used.")
 	}
 }
+
+func TestFormatRateLimitResetDuration(t *testing.T) {
+	d := 120*time.Minute + 12*time.Second
+	got := formatRateLimitResetDuration(d)
+	want := "[rate reset in 120m12s]"
+
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
+	}
+
+	d = 14*time.Minute + 2*time.Second
+	got = formatRateLimitResetDuration(d)
+	want = "[rate reset in 14m02s]"
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
+	}
+
+	d = 2*time.Minute + 2*time.Second
+	got = formatRateLimitResetDuration(d)
+	want = "[rate reset in 2m02s]"
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
+	}
+
+	d = 12 * time.Second
+	got = formatRateLimitResetDuration(d)
+	want = "[rate reset in 12s]"
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
+	}
+
+	d = -1 * (2*time.Hour + 2*time.Second)
+	got = formatRateLimitResetDuration(d)
+	want = "[rate limit was reset 120m02s ago]"
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
```

Comment:
Thank you so much for noticing that. 
Just came from the another language world where all the time I've seen "Expected: bla-bla, but got: bla-bla-bla".

Improved my code :)